### PR TITLE
Cfd 69 app splash screen

### PR DESCRIPTION
**Pull request by nm-le**
### Reason for Change

Fixes CFD-

Completes CFD-69

Contributes to CFD-

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other

### Testing

- [ ] Test A
- [ ] Test B

### CODE REVIEW

File: .gitignore
Comment (BakerWJ) [inline .gitignore:1]: revert

File: .vscode/settings.json
Comment (BakerWJ) [inline .vscode/settings.json:1]: revert

File: apps/expo/assets/landing_page.svg
Comment (BakerWJ) [inline apps/expo/assets/landing_page.svg:1]: can you use the updated hi-fi svg from https://www.figma.com/file/AWQNOMKvFlxKCjmbfuPdS4/Wireframes?type=design&node-id=351-542&mode=design&t=i6dT13lZQCwQ5nxG-0

File: apps/expo/package.json [hunk 1]
```diff
@@ -54,7 +55,7 @@
     "@cfd/prettier-config": "^0.1.0",
     "@cfd/tailwind-config": "^0.1.0",
     "@cfd/tsconfig": "^0.1.0",
-    "@expo/config-plugins": "^7.2.5",
+    "@expo/config-plugins": "~7.2.2",
```
Comment (BakerWJ) [inline apps/expo/package.json:58]: why did this get changed?

File: apps/expo/src/app/index.tsx
Comment (BakerWJ) [inline apps/expo/src/app/index.tsx:1]: should the splash screen be placed in the layout.tsx file instead?

File: apps/expo/src/app/index.tsx [hunk 2]
```diff
@@ -19,22 +23,61 @@ const Index = () => {
...
+    flex: 1,
+    alignItems: "center",
+    justifyContent: "center",
+  },
+  splashImage: {
+    width: 500,
+    height: 700,
+  },
+});
+
```
Comment (BakerWJ) [inline apps/expo/src/app/index.tsx:82]: use tailwind for styling

**Review by BakerWJ — CHANGES_REQUESTED** [review]

File: apps/expo/assets/landing_page.svg
Comment (BakerWJ) [inline apps/expo/assets/landing_page.svg:1]: how was this svg generated -- looks like it has a weird blue artifact at the top of the logo

File: .vscode/settings.json
Comment (BakerWJ) [inline .vscode/settings.json:1]: revert

File: .gitignore
Comment (BakerWJ) [inline .gitignore:1]: revert

File: apps/expo/src/custom.d.ts
Comment (BakerWJ) [inline apps/expo/src/custom.d.ts:1]: is this still necessary?

File: apps/expo/package.json [hunk 3]
```diff
@@ -43,6 +43,7 @@
     "react-native-root-toast": "^3.5.1",
     "react-native-safe-area-context": "4.6.3",
     "react-native-screens": "~3.22.1",
+    "react-native-svg": "13.9.0",
```
Comment (BakerWJ) [inline apps/expo/package.json:46]: is this still being used?

File: apps/expo/assets/landing_page.svg
Comment (nm-le) [reply apps/expo/assets/landing_page.svg:1]: uhh i got from the latest figma
image

File: apps/expo/src/custom.d.ts
Comment (nm-le) [reply apps/expo/src/custom.d.ts:1]: nope I will remove

File: apps/expo/package.json (hunk 3 above)
Comment (nm-le) [reply apps/expo/package.json:46]: No, but should I leave it for other to use?

Comment (BakerWJ) [reply apps/expo/package.json:46]: you can revert it, and someone else can add it if they're using it

File: apps/expo/package.json
Comment (BakerWJ) [inline apps/expo/package.json:1]: revert

File: pnpm-lock.yaml
Comment (BakerWJ) [inline pnpm-lock.yaml:1]: revert

**Review by BakerWJ — CHANGES_REQUESTED** [review]
make sure that these  2 files don't show any changes in the PR details

**Comment by nm-le** [comment]
> make sure that these 2 files don't show any changes in the PR details

image

config-plugins seems to need different version according to lint check?

**Comment by BakerWJ** [comment]
> > make sure that these 2 files don't show any changes in the PR details
> 
> 
> 
> image
> 
> 
> 
> config-plugins seems to need different version according to lint check?

That's because you still have some changes in pnpm-lock.yaml that need to be reverted

**Review by BakerWJ — APPROVED** [review]

**Review by BakerWJ — APPROVED** [review]

**Pull request merged**